translations/w3c_website_templates_bundle+intl-icu.en.yaml

```
@@ -191,3 +191,34 @@ events:
     website: Website
 alumni:
     date_range: "{start} to {end}"
+error:
+    401:
+        title: Authentication Required
+        message: >-
+            <p>A W3C account is required to access the page that you requested.</p>
+            <ul>
+            <li>If you don't have one, <a href="https://www.w3.org/accounts/request">request a new account</a>.</li>
+            <li>If you have an account but forgot your password, <a href="https://www.w3.org/accounts/recover">recover your password</a>.</li>
+            <li>In case of trouble with your W3C account, please send an email to
+            <a href="mailto:sysreq@w3.org">sysreq@w3.org</a>.
+            </li>
+            </ul>
+    403:
+        title: Access Denied
+        message: >-
+            <p>You do not have sufficient privileges to access the page that
+            you requested: <strong>{message}</strong></p>
+    404:
+        title: Page Not Found
+        message: >-
+          <p>Sorry, the page you were trying to reach does not exist. Here are some steps you may try to find the page you were looking for:</p>
+          <ul>
+          <li>If you typed the URL by hand then please make sure that it is exactly as it should be.</li>
+          <li>If you are looking for information on a particular subject, please start on the <a href="/">W3C home page</a>.</li>
```

using `https://www.w3.org/`instead of `/` as we will be using those error pages on different hostnames.

```suggestion
          <li>If you are looking for information on a particular subject, please start on the <a href="https://www.w3.org/">W3C home page</a>.</li>
```